paimon-web-api/src/main/java/org/apache/paimon/web/api/action/MysqlSyncTableActionService.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.api.action;
+
+import org.apache.paimon.web.api.action.context.ActionContext;
+
+import com.google.auto.service.AutoService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Mysql sync table action service . */
+@AutoService(ActionService.class)
```

Comment:
Why does this use auto service of google?

Reply:
The intention is to make this feature pluggable, enabling users to upload their own JAR files in the future and thereby implement custom Action tasks according to their specific needs.